MatrixSDK/Data/RoomList/CoreData/MXCoreDataRoomListDataFetcher.swift

```diff
@@ -94,14 +99,18 @@ internal class MXCoreDataRoomListDataFetcher: NSObject, MXRoomListDataFetcher {
             properties.append(property)
         }
         request.propertiesToFetch = properties
+        request.resultType = .dictionaryResultType
```

Comment:
Thinking about it, might also be worth adding a comment here to say that it fixes issues that were seen on iOS 12 for future reference.